telos-frontend/src/components/todo/JournalTodo.jsx

```diff
@@ -261,6 +249,8 @@ const JournalTodo = ({ data, date }) => {
         })}
       </List>
       <div>
+       { /* A textbox to indicate to the user that they can enter new to do's.
```

Comment:
Here, a little bit change is needed.

Reply:
Hey Evan, thank you for the suggestion! I have removed the space inside the comment '  { /*  ' as the following change after your feedback :)